### PR TITLE
 Make the reboot required text more visible in the console output

### PR DESCRIPTION
**Pull request by matejmatuska**
The "A reboot is required to continue. Please reboot your system." message is printed before the reports summary and thus is easily overlooked by users.

This patch adds a second such message after the report summary to improve this.

Example (slightly outdated, the extra comma is already removed):
image

Jira: RHEL-22736

### CODE REVIEW

**Comment by matejmatuska** [comment]
The exact message/wording is up for discussion of course, but I think it should also mention that the reboot can/will take more time than normal reboot.
The text could also be bold/colored, but that might be too much in my opinion.

File: commands/upgrade/__init__.py [hunk 1]
```diff
@@ -113,6 +113,11 @@ def upgrade(args, breadcrumbs):
 
     if workflow.failure:
         sys.exit(1)
+    elif not args.resume:
+        sys.stdout.write(
+            'To continue with the upgrade reboot the system.'
```
Comment (fernflower) [inline commands/upgrade/__init__.py:118]: Nit: I'd rephrase it `Please reboot the system to continue with the upgrade` to emphasize the reboot action necessity.

**Comment by pirat89** [comment]
@matejmatuska add please jira ticket number to the commit msg and PR description.

**Comment by fernflower** [comment]
Rebase against master should fix the tests

File: commands/upgrade/__init__.py (hunk 1 above)
Comment (pirat89) [reply commands/upgrade/__init__.py:118]: I like the suggested wording, I would just drop the please, so

```suggestion
            'Reboot the system to continue with the upgrade.'
```

but I am also ok with the current one. I will ask Chris about his opinion.

Comment (pirat89) [reply commands/upgrade/__init__.py]: @matejmatuska FYI,  I've updated the first sentence as above + rebased.

File: commands/upgrade/__init__.py [hunk 2]
```diff
@@ -113,6 +113,11 @@ def upgrade(args, breadcrumbs):
 
     if workflow.failure:
         sys.exit(1)
+    elif not args.resume:
+        sys.stdout.write(
+            'Reboot the system to continue with the upgrade.'
+            ' This might take a while depending on the system configuration.\n'
```
Comment (fernflower) [inline commands/upgrade/__init__.py:119]: Nit: Could we please not start the next sentence with a whitespace, but rather finish the previous one with it? Other messages cling to that style and imho it's easier to read through.

`Reboot the system to continue with the upgrade. '`
`'This might take a while depending on the system configuration.\n'`

**Review by fernflower — COMMENTED** [review]
One nit that is triggering my ocd :)

**Review by pirat89 — APPROVED** [review]
```
============================================================
                   END OF REPORT OVERVIEW                   
============================================================

Answerfile has been generated at /var/log/leapp/answerfile
Reboot the system to continue with the upgrade. This might take a while depending on the system configuration.
```

Tested, the msg is printed only when the system should be rebooted
* `preupgrade` - no msg
* `upgrade` with inhibitor or error - no msg
* `upgrade --resume (after the reboot) - no msg
* `upgrade` without inhibitor or error - msg printed
